src/ws-header/simple-steam.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * This utility class implements a simple stream to listen continuously for changes
+ */
+export class SimpleSteam {
```

Comment:
I can't get my hands fully around this simple stream, I see you use it for listening to changes in localstorage but not sure why is that.
* Rename it to "Stream" not "Steam" heheheh